app/src/main/res/layout/admin_auth_activity.xml

```diff
@@ -27,10 +27,10 @@
       <androidx.appcompat.widget.Toolbar
         android:id="@+id/admin_auth_toolbar"
         android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
+        android:layout_height="wrap_content"
         android:background="@color/component_color_admin_auth_activity_toolbar_color"
         android:fontFamily="sans-serif"
-        android:minHeight="?attr/actionBarSize"
+        android:minHeight="48dp"
```

Comment:
Error can be fixed by only making this change

`android:layout_height="wrap_content"`

Reply:
Done